README.md

```diff
@@ -2,8 +2,6 @@
 
 List of student submissions for the [WWDC 2020 Swift Student Challenge](https://developer.apple.com/wwdc20/swift-student-challenge/).
 
-Check out the [YouTube Playlist](https://www.youtube.com/playlist?list=PLZw7eGQJuMjlEcsJ0CgYtxQHgrw1t8aSc) to watch the submissions.
```

Comment:
Your branch is out of date, and is removing some items that exist. Can you make sure it is merged with master?